pymc/distributions/continuous.py

```diff
@@ -3420,11 +3417,11 @@ class Rice(PositiveContinuous):
 
     Parameters
     ----------
-    nu: float
+    nu : tensor_like of float
         noncentrality parameter.
-    sigma: float
+    sigma : tensor_like of float, default 1
         scale parameter.
-    b: float
+    b : tensor_like of float
```

Comment:
I think both `nu` and `b` are optional in the sense that one of them is required, but only one of them is so using `b` without `nu` is valid (making `nu` optional) and viceversa. 

I'd add `, optional` and then on the description on `nu` saying like "When not provided, b is required" or "Required if b is not provided". Same for b

Reply:
done : )